compiler/test/grainfmt/comments.expected.gr

```diff
@@ -1,3 +1,9 @@
+// Comment before first attr
+/* Block comment before attr */@noPervasives // Line comment on attr
```

Comment:
This is wrong. It needs to have a space between `*/` and `@`

Reply:
Looks like in other places it actually puts the block comment on its own line, which I think I prefer here